fastssl/models/barlow_twins.py

```diff
@@ -33,6 +33,126 @@ def BarlowTwinLoss(model, inp, _lambda=None):
     inp = list(inp)
     _ = inp.pop(1)
     num_augs = len(inp)
+    # for x in inp:
```

Comment:
let's clean this up soon 😅

Reply:
haha I will merge it for now. We should do another PR for cleaning up code!